src/script/user/UserRepository.ts

```diff
@@ -482,10 +459,8 @@ export class UserRepository {
       }
     };
 
-    const chunksOfUserIds = chunk<string | QualifiedId>(userIds, Config.getConfig().MAXIMUM_USERS_PER_REQUEST);
-    const resolveArray = await Promise.all(
-      chunksOfUserIds.map(userChunk => getUsers(userChunk as string[] | QualifiedId[])),
-    );
+    const chunksOfUserIds = chunk<QualifiedId>(userIds, Config.getConfig().MAXIMUM_USERS_PER_REQUEST);
+    const resolveArray = await Promise.all(chunksOfUserIds.map(userChunk => getUsers(userChunk)));
```

Comment:
does this work also? 

```suggestion
    const resolveArray = await Promise.all(chunksOfUserIds.map(getUsers));
```

Reply:
LGTM